tools/dump_tool/ramdumpParser.py

```diff
@@ -196,9 +198,9 @@ def parse_file(self,log_file,debug=False):
 							print(' ')
 							print(' ')
 
-						# If the PC is not withing RAM range, it's a Prefetch issue
+						# If the PC is not withing RAM text range, it's a Prefetch issue
 						# So, fill the PC with LR and help to get the call stack
-						if ( pc < g_stext or pc > g_etext):
+						if (pc < g_stext_flash or pc > g_etext_flash) or (pc < g_stext_ram or pc > g_etext_ram):
```

Comment:
Maybe it should be `and`.
```
if (pc < g_stext_flash or pc > g_etext_flash) and (pc < g_stext_ram or pc > g_etext_ram):
```
for example, pc is in the range of flash text region and flash text region is located before ram text region.
pc < g_stext_flash : false
pc < g_etext_flash : true
pc < g_stext_ram : false
pc < g_etext_ram : false
=> (false or true) and (false or false) => false

Reply:
In order to avoid confusion, i think it is better to use our original check with a not condition as below:
`if (!(pc >= stext_flash & pc <= etext_flash) && !(pc >= stext_ram && pc <= etext_ram))`

Reply:
Agreed. 
Updated the code